tests/e2e/reports/reports-sidebar-filter.wdio-spec.js

```diff
@@ -0,0 +1,70 @@
+const moment = require('moment');
+
+const utils = require('../../utils');
+const commonElements = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const userFactory = require('../../factories/cht/users/users');
+const placeFactory = require('../../factories/cht/contacts/place');
+const personFactory = require('../../factories/cht/contacts/person');
+const reportFactory = require('../../factories/cht/reports/generic-report');
+
+describe('Reports Sidebar Filter', () => {
+  let savedReports;
+  const today = moment();
+  const parent = placeFactory.place().build({ _id: 'dist1', type: 'district_hospital' });
+  const user = userFactory.build({ username: 'john', roles: [ 'chw' ] });
```

Comment:
Same in here, you could remove the parameter sent to the `userFactory.build()`, it will create by default a user named "user1" with the role "chw"